src/main/java/edu/wpi/first/pathweaver/CreateProjectController.java

```diff
@@ -96,14 +89,15 @@ private void initialize() {
     ObservableList<TextField> allFields = FXCollections.observableArrayList(numericFields);
     allFields.add(directory);
 
-    timeControls = FXCollections.observableArrayList(timeLabel, timeStep, timeUnits);
-    velocityControls = FXCollections.observableArrayList(velocityLabel, maxVelocity, velocityUnits);
-    accelerationControls = FXCollections.observableArrayList(accelerationLabel, maxAcceleration, accelerationUnits);
-    jerkControls = FXCollections.observableArrayList(jerkLabel, maxJerk, jerkUnits);
-    wheelBaseControls = FXCollections.observableArrayList(wheelBaseLabel, wheelBase, wheelBaseUnits);
+    var timeControls = List.of(timeLabel, timeStep, timeUnits);
+    var velocityControls = List.of(velocityLabel, maxVelocity, velocityUnits);
+    var accelerationControls = List.of(accelerationLabel, maxAcceleration, accelerationUnits);
+    var jerkControls = List.of(jerkLabel, maxJerk, jerkUnits);
+    var wheelBaseControls = List.of(wheelBaseLabel, wheelBase, wheelBaseUnits);
 
+    var allTooltipControls = new ArrayList<Control>();
```

Comment:
This could be

```java
List<Control> allTooltipControls = Stream.of(timeControls, velocityControls, accelerationControls, jerkControls, wheelBaseControls)
    .flatMap(List::stream)
    .collect(Collectors.toList());
```

And since this variable only seems to be used once later in this method, perhaps the stream could be inlined there:

```java
Stream.of(timeControls, velocityControls, accelerationControls, jerkControls, wheelBaseControls)
    .flatMap(List::stream)
    .forEach(control -> control.getTooltip().setShowDelay(Duration.millis(300)));
```